content/hi/docs/reference/glossary/api-eviction.md

```diff
@@ -0,0 +1,26 @@
+---
+title: API-
+id: api-eviction
+date: 2021-04-27
+full_link: /docs/concepts/scheduling-eviction/api-eviction/
+short_description: >
+API-eviction निष्कासन वह प्रक्रिया है जिसके द्वारा आप एक बनाने के लिए निष्कासन API का उपयोग करते हैं
```

Comment:
Just noticed that the [Netlify build](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65204e17ca441a00080b6be4) didn't succeed due to an error in Line 7, as indicated in the error message below:

> 7:13:19 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
> 7:13:19 PM: Error: Error building site: "/opt/build/repo/content/hi/docs/reference/glossary/api-eviction.md:7:1": failed to unmarshal YAML: yaml: line 7: could not find expected ":"

Additionally, when viewing the changed [Markdown file](https://github.com/shelar1423/website/blob/main/content/hi/docs/reference/glossary/api-eviction.md) in the GitHub user interface, there seems to be a parsing error, as indicated by the message:

> Error in user YAML: (<unknown>): could not find expected ':' while scanning a simple key at line 6 column 1

It seems that the missing spaces might be causing the errors. Try below suggestion:

```suggestion
short_description: >
  API-eviction निष्कासन वह प्रक्रिया है जिसके द्वारा आप एक बनाने के लिए निष्कासन API का उपयोग करते हैं
```

Reply:
sure ,implemented it